localization/simp_chinese/newsfeed_l_simp_chinese.yml

```
@@ -68,20 +68,20 @@
  newsfeed_war_lost: "[GetPlayer.Custom2('newsfeed_relation_or_me', SCOPE.sC('protagonist'))|U]输掉了[war|E]"
  newsfeed_war_lost_desc: "[SCOPE.sC('protagonist').GetUIName|U]在[SCOPE.sWar('war').GetName]中输给了[GetPlayer.Custom2('newsfeed_relation_name_or_me', SCOPE.sC('antagonist'))]"
  newsfeed_war_white_peace: "[GetPlayer.Custom2('newsfeed_relation_or_me', SCOPE.sC('protagonist'))|U]谈判达成了[white_peace|E]"
- newsfeed_war_white_peace_desc: "[SCOPE.sC('protagonist').GetUIName|U]以无条件和平结束了与[GetPlayer.Custom2('newsfeed_relation_name_or_me', SCOPE.sC('antagonist'))]的[SCOPE.sWar('war').GetName]"
+ newsfeed_war_white_peace_desc: "[SCOPE.sC('protagonist').GetUIName|U]以无条件和平结束了与[GetPlayer.Custom2('newsfeed_relation_name_or_me', SCOPE.sC('defender'))]的[SCOPE.sWar('war').GetName]"
```

This should stay `'antagonist'`. This was originally a mistake on my part that got carried over - sorry about that!

```suggestion
 newsfeed_war_white_peace_desc: "[SCOPE.sC('protagonist').GetUIName|U]以无条件和平结束了与[GetPlayer.Custom2('newsfeed_relation_name_or_me', SCOPE.sC('antagonist'))]的[SCOPE.sWar('war').GetName]"
```